.storybook/main.ts

```diff
@@ -0,0 +1,27 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-mdx-gfm'],
+  features: {
+    buildStoriesJson: true,
+  },
+  previewHead: (head) => `
+    ${head}
+    <link rel="stylesheet" type="text/css" title="themeCss" href="styles/core-theme.css" />
+  `,
+  stories: [
+    '../packages/design-system/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../packages/ds-healthcare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../packages/ds-medicare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+  ],
+  staticDirs: ['./static/', '../packages/design-system/src'],
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
```

Comment:
Does this unset options, wondering if this line can be removed?

Reply:
I tried removing it but it appears to be required - even though it's empty. [SB docs](https://storybook.js.org/docs/react/configure/overview#configure-your-storybook-project) show it as empty as well.